openmls/src/treesync/node.rs

```diff
@@ -11,11 +12,27 @@ pub(crate) mod leaf_node;
 pub(crate) mod parent_node;
 
 /// Container enum for leaf and parent nodes.
-#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+///
+/// ```c
+/// // draft-ietf-mls-protocol-17
+/// struct {
+///     NodeType node_type;
+///     select (Node.node_type) {
+///         case leaf:   LeafNode leaf_node;
+///         case parent: ParentNode parent_node;
+///     };
+/// } Node;
+/// ```
+#[derive(
+    Debug, PartialEq, Eq, Clone, Serialize, Deserialize, TlsSize, TlsDeserialize, TlsSerialize,
+)]
 #[allow(missing_docs)]
 #[allow(clippy::large_enum_variant)]
```

Comment:
My local clippy is not complaining if I remove those two allows. While you're touching this code, could you check if we can get rid of them?

Reply:
Removing code is always good :-)